frontend/src/Dashboard/Header/Login.js

```diff
@@ -28,25 +28,15 @@ class Login extends Component {
     .then(({ data }) => {
       console.log('got data', data);
       localStorage.setItem('jwt_user_token', data.authenticate.jwtToken);
-      this.setState({ redirectToReferrer: true })
+      window.location.reload();
```

Comment:
Can you explain why you had to do this here? Looking at MDN, it sounds like we have to force a reload of server content?
https://developer.mozilla.org/en-US/docs/Web/API/Location/reload

Reply:
It should be possible to do this through the router instead of using location.reload().

As far as the server context goes, since we set the jwt in localStorage a cache reload should be just fine.

Reply:
Got it. Up to you, but maybe its worth adding low priority backlog ticket (Asana or Github Issues) to investigate a React Router v4 approved way to do this?

Reply:
The reason I suggest Github Issues for that is because this would be a pretty discreet place for a volunteer contribution. It would be nice if we moved towards having a public list of bite size places people can help. Something low priority like this might just become clutter in Asana.

Reply:
Created issue #37